index.html

```diff
@@ -15,11 +15,10 @@
     <title>Gitsub</title>
   </head>
   <body>
-
     <nav class="navbar navbar-expand-lg navbar-dark">
       <ul class="navbar-nav">
         <li class="nav-item">
-          <a href="#" class="nav-link">Overview</a>
+          <a href="index.html" class="nav-link">Overview</a>
         </li>
```

Comment:
Shouldn't this link to overview.html?

Reply:
I was under the impression we were treating the index.html as the overview page.

Reply:
Unless I am mistaken!